account_balance_query_test.go

```diff
@@ -9,11 +9,22 @@ import (
 func TestAccountBalanceQuery_Execute(t *testing.T) {
 	env := NewIntegrationTestEnv(t)
 
-	_, err := NewAccountBalanceQuery().
+	balance, err := NewAccountBalanceQuery().
+		SetAccountID(env.OriginalOperatorID).
+		SetNodeAccountIDs(env.NodeAccountIDs).
+		Execute(env.Client)
+	assert.NoError(t, err)
+
+	println(balance.Hbars.String())
+
+	_, err = NewAccountBalanceQuery().
 		SetAccountID(env.OperatorID).
 		SetNodeAccountIDs(env.NodeAccountIDs).
 		Execute(env.Client)
 	assert.NoError(t, err)
+
+	err = CloseIntegrationTestEnv(env, nil)
```

Comment:
I'd recommend you simply pass in `t` into `CloseIntegrationTestEnv` so it can automatically do the check.